src/command/info/info.c

```diff
@@ -640,7 +640,7 @@ stanzaInfoList(List *stanzaRepoList, const String *backupLabel, unsigned int rep
         VariantList *repoSection = varLstNew();
 
         int stanzaStatusCode = -1;
-        unsigned int stanzaCipherType = 0;
+        uint64_t stanzaCipherType = 0;
```

Comment:
I'm debating whether this should be `= cipherTypeNone` instead of `= 0`. We should never get to this function if there are no repositories configured so it's probably mute, but the intent was this would be the value of None which meant if we ever got to line 748 without going through the FOR loop at line 649 (which we won't), then this would be the right type. I know the value is overloaded and this was not the best way to go about this, but...

Reply:
No objections so done in c0d8f7eb649c4.